src/Uno.UWP/Storage/StorageFolder.cs

```diff
@@ -176,42 +176,44 @@ internal StorageFolder(string name, string path)
 
 		public IAsyncOperation<StorageFile> CreateFileAsync(string desiredName) => CreateFileAsync(desiredName, CreationCollisionOption.FailIfExists);
 
-        public IAsyncOperation<StorageFile> CreateFileAsync(string path, CreationCollisionOption option) =>
+        public IAsyncOperation<StorageFile> CreateFileAsync(string desiredName, CreationCollisionOption options) =>
             AsyncOperation.FromTask(async ct =>
             {
-                if (File.Exists(global::System.IO.Path.Combine(Path, path)))
+                if (File.Exists(global::System.IO.Path.Combine(Path, desiredName)))
                 {
-                    switch (option)
+                    switch (options)
                     {
+						case CreationCollisionOption.FailIfExists:
+							throw new Exception("Cannot create a file when that file already exists.");
```

Comment:
Codacy found an issue: ['System.Exception' should not be thrown by user code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5685225)